scripts/ci/validate-wallet-swagger.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+if [[ $TRAVIS_COMMIT == "" ]]; then
+    echo "TRAVIS_COMMIT is unspecified"
+    exit 1
+fi
+
+spec_path="https://raw.githubusercontent.com/input-output-hk/cardano-sl/$TRAVIS_COMMIT/wallet-new/spec/swagger.json"
```

Comment:
Very cool, this will also allow us to reference to this version externally in `editor.swagger.io`, which would be a boon in reviewing new API endpoints :)